### PR TITLE
Create presubmit.yml for Bazel continuous integration

### PR DESCRIPTION
**Pull request by achew22**
This (+ a PR in https://github.com/bazelbuild/continuous-integration) will put
make Bazel changes that break Kythe visible to the Bazel team before a
release is cut.

This just builds for Ubuntu 18.04, but can be expanded to do windows,
osx, or ubuntu 16.04. For now I want to start small and see if we can get
the two to just talk to each other.

### CODE REVIEW

File: presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+tasks:
+  ubuntu_1804:
+    platform: ubuntu1804
+    build_targets:
+    - "..."
```
Comment (shahms) [inline presubmit.yml]: Both of these should probably be "//kythe/..." rather than just "..."

Comment (shahms) [reply presubmit.yml]: Also, from the documentation shouldn't this be: `.bazelci/presubmit.yml`?

Comment (achew22) [reply presubmit.yml]: Done

**Comment by achew22** [comment]
Friendly ping

**Pull request merged**